manual/python/index.md

```diff
@@ -615,14 +630,14 @@ modifier methods on the `std::string` objects.
 
 - When obtaining the boolean value of a C++ instance proxy, both old and new PyROOT return
 `False` when such proxy points to null. On the other hand, when the proxy points to a C++ object,
-old PyROOT just returns `True`, while new PyROOT has slightly modified this behaviour: in new
+old PyROOT just returns `True`, while new PyROOT has slightly modified this behaviour. In new
```

Comment:
Same comment here about the `:`, perhaps it's a language difference? I use `:` to link with a clarification/further explanation of my previous statement.